### PR TITLE
Refactor the command reset to third-level commands, like: keamd reset

### PR DESCRIPTION
**Pull request by luomengY**
cloud and keadm reset edge. If uses keadm rest, both the cloud and the edge will be reset.`

**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
image
image

**Which issue(s) this PR fixes**:

Fixes : A part of issue  https://github.com/kubeedge/kubeedge/issues/5317

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/reset_others.go [hunk 1]
```diff
@@ -116,108 +116,55 @@ func NewKubeEdgeReset() *cobra.Command {
...
 			}
 
 			// 3. Clean stateful directories
-			if err := cleanDirectories(isEdgeNode); err != nil {
+			if err := util.CleanDirectories(isEdgeNode); err != nil {
 				return err
 			}
 
-			// cleanup mqtt container
-			if err := RemoveMqttContainer(reset.Endpoint, ""); err != nil {
```
Comment (WillardHu) [inline keadm/cmd/keadm/app/cmd/reset_others.go:129]: Maybe we still need to consider the case of removing mqtt.

File: keadm/cmd/keadm/app/cmd/edge/reset_other.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+)
+
+var (
+	resetLongDescription = `
+keadm reset edge command can be executed edge node
+In edge node it shuts down the edge processes of KubeEdge
+`
+	resetExample = `
+For edge node edge:
+keadm reset
```
Comment (WillardHu) [inline keadm/cmd/keadm/app/cmd/edge/reset_other.go:25]: keadm reset edge

**Review by WillardHu — CHANGES_REQUESTED** [review]

File: keadm/cmd/keadm/app/cmd/cloud/reset.go [hunk 3]
```diff
@@ -0,0 +1,103 @@
+package cloud
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/cloud/reset.go:1]: Add the License header

**Review by fisherxu — COMMENTED** [review]
/approve
/assign @WillardHu

**Pull request merged**